checks/microbenchmarks/mpi/osu/osu_tests.py

```diff
@@ -37,6 +37,8 @@ class cscs_osu_pt2pt_check(cscs_osu_benchmarks):
     valid_systems = ['daint:gpu', 'daint:mc', 'dom:gpu', 'dom:mc',
                      'eiger:mc', 'pilatus:mc', 'arolla:cn', 'tsa:cn']
     valid_prog_environs = ['PrgEnv-gnu']
+    if self.current_system.name in ('arolla', 'tsa'):
+        num_iters = 100
```

Comment:
This will not work, you can do it in a post-init hook though:
```python
    @run_after('init')
    def set_number_of_iterations(self):
        if self.current_system.name in ('daint'):
            self.num_iters = 100
```

Reply:
Thanks.